action/src/index.ts

```diff
@@ -0,0 +1,63 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+import * as childProcess from 'child_process';
+import * as fs from 'fs';
+
+/**
+ * writeFileToConsole contents of file to console.
+ * @param {string} path - filepath to write to the console
+ */
+function writeFileToConsole(path: string) {
+  try {
+    const data = fs.readFileSync(path, 'utf8');
+    console.log(`${path}: ${data}`);
+  } catch (e) {
+    console.log(`$unable to read {path}, skipping: ${e}`);
+  }
+}
+
+/**
+ * Run a specified command.
+ * @param {string} cmd - command to run
+ */
+function runCmd(cmd: string) {
+  try {
+    childProcess.execSync(cmd);
+  } catch (error) {
+    writeFileToConsole('serverlog_stdout.txt');
+    writeFileToConsole('serverlog_stderr.txt');
+    writeFileToConsole('function_output.json');
+    throw error;
+  }
+}
+
+async function run(): Promise<void> {
+  try {
```

Comment:
I think we can remove the try/catch block if we don't throw in the `runCmd` function above.